scripts/experiments_images/GANomaly.jl

```diff
@@ -77,11 +78,11 @@ function fit(data, parameters)
 		fit_t = fit_t,
 		model = (info[2]|>cpu, info[3]|>cpu),
 		history = info[1], # losses through time
-        npars = info[4], # number of parameters
-        iters = info[5] # optim iterations of model
+		npars = info[4], # number of parameters
+		iters = info[5] # optim iterations of model
 		)
 
-	return training_info, [(x -> GenerativeAD.Models.anomaly_score_gpu(generator|>cpu, x; dims=3), parameters)]
+	return training_info, [(x -> GenerativeAD.Models.anomaly_score_gpu(generator|>cpu, x; dims=3)[:], parameters)]
```

Comment:
Is this a cpu or gpu array?

Reply:
`|> cpu` is probably useless here. The function is written to work on GPU (with limited memory -> batches), but when computer does not have GPU it runs on cpu instead. 

but it always returns cpu array